CactbotOverlay/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,4 @@
 // - Revision
 // GitHub has only 3 version components, so Revision should always be 0.
 [assembly: AssemblyVersion("0.10.9.0")]
-[assembly: AssemblyFileVersion("0.10.9.0")]
\ No newline at end of file
+[assembly: AssemblyFileVersion("0.10.9.0")]
```

Comment:
Probably worth removing this file from the set of changes; it's tedious to keep it up to date for the sake of a final new line and potentially dangerous to commit with the wrong version number.

Reply:
It's actually more work to remove this change than it is to keep the version number updated since rebase automatically takes care of that. I'll still remove it since it wasn't an intentional change to begin with.